README.md

```diff
@@ -1,13 +1,11 @@
-# Project Todos
+# Project Todos 🖊️
 
-Replace this readme with your own information about your project.
-
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+In this week's project I've created a to do app using Redux. It was quite a challenge with this programming langugage I believe, but I had so much fun while doing it!
 
 ## The problem
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+I've discussed the theory about Redux with the people in my class and I've used the live sessions well to be able to grasp the concept as much as possible. I also watched some good youtubers explaining Redux. Looking forward to continue learning more!
 
 ## View it live
 
-Every project should be deployed somewhere. Be sure to include the link to the deployed project so that the viewer can click around and see what it's all about.
+https://dothistoday.netlify.app/
```

Comment:
Your page looks really great and I like that you used a startpage to begin with. I like how you kept it so clean in regards to colors and styling and rather put more focus to logic in this project. The functionality is working well, the only thing I noticed when clicking/adding more todo's is that the date (time) will update for each added todo. For example: if you add one todo at 9:00 am this todo will update its time with another added todo so the two of them, after adding the second one at say 9:01 am, will have time 9:01 am. I actually struggled a bit with this too and I did not manage to solve but I can imagine we somehow need to save the particular date within each todo-item. 

All in all, I love your to-do list and I should not forgot to mention the cute + relevant images you added! :)

Reply:
Thank you Camilla!! About the timestamp, I asked in S.O and got this answer from Matilda - will try it out later: https://stackoverflow.com/c/technigo/questions/3717

Reply:
Ah, nice! I'm bookmarking this so I can try it out too! Thanks! :)